### ISSUE TITLE
maven building failed in Ubuntu 18.06+

### ISSUE THREAD

**Issue by qrafzv** [GCRC/nunaliit#777]
Credit to Rob, mvn building failed in Ubuntu 18.06 and 18.10
surefire report:
`java.awt.AWTError: Assistive Technology not found: org.GNOME.Accessibility.AtkWrapper`
According to these links:

https://askubuntu.com/questions/695560/assistive-technology-not-found-awterror
https://stackoverflow.com/questions/17727157/imageio-not-working-on-ubuntu
https://github.com/Microsoft/vscode-arduino/issues/644

Two known ways to solve this issue

First: install openjdk-8-jdk (instead of openjdk-8-jdk-headless)

Second: editing the accessibility.properties file for OpenJDK:
sudo vim /etc/java-8-openjdk/accessibility.properties
Comment out the following line:
assistive_technologies=org.GNOME.Accessibility.AtkWrapper
Running on OpenJDK 11 the file is
/etc/java-11-openjdk/accessibility.properties
@ahayes

**Comment by qrafzv** [comment]
By the way, introducing the imageio api causes this issue,
To be specific, start from commit 2a478fe

**Comment by ahayes** [comment]
Depends on #819

**Comment by ahayes** [comment]
Today one of our servers updated to version `8u402-ga-2ubuntu1~18.04_amd64` of the openjdk-8 headless packages and the default accessibility.properties files has the `assistive_technologies=org.GNOME.Accessibility.AtkWrapper` line commented out with "Doesn't work, see LP: #935296" as a comment. That refers to [issue number 935296 in the Ubuntu LaunchPad system](https://bugs.launchpad.net/ubuntu/+source/openjdk-6/+bug/935296).

I guess we were ahead of our time. :)
